runtime/compiler/control/J9Options.cpp

```diff
@@ -368,7 +368,8 @@ char * J9::Options::_externalOptionStrings[J9::ExternalOptions::TR_NumExternalOp
    "-XX:-JITServerAOTCachePersistence",   // = 64
    "-XX:JITServerAOTCacheDir=",           // = 65
    "-XX:JITServerAOTCacheName=",          // = 66
-   // TR_NumExternalOptions                  = 67
+   "-XX:codecachetotalRAMPercentage=",    // = 67
```

Comment:
Suggest `-XX:codecachetotalMaxRAMPercentage=` because if there is plenty of RAM (say 16 GB) we allocate only 256MB and not 50% of 16 GB.

Reply:
Done.